packages/api/.env.local.sample

```diff
@@ -4,9 +4,8 @@
 
 # ---------- PRIVATE Keys ----------
 # All keys here a base local development public keys so you can use them to test the app locally and should not be considered a security risk.
-ETHERS_DEPLOYER_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-ETHERS_OPERATOR_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
hi @jplodge-pro - did you mean to also remove the Operator key?  i'm still investigating, but what i found is VaultFactory.createVaults needs to be called by the VaultFactory operator.  this is not the same as the Deployer.

Reply:
Hi @lucasia 
Yes, these were the edits as worked out with @Krishnakumarskr 
If the key is still required, we will obviously restore it. Also, Issue Accepted!